pages/common/tar.md

```diff
@@ -4,34 +4,34 @@
 > Often combined with a compression method, such as gzip or bzip.
 > More information: <https://www.gnu.org/software/tar>.
 
-- Create an archive from files:
+- [C]reate an archive from [f]iles:
 
 `tar cf {{target.tar}} {{file1}} {{file2}} {{file3}}`
 
-- Create a gzipped archive:
+- [C]reate a g[z]ipped archive from [f]iles:
 
 `tar czf {{target.tar.gz}} {{file1}} {{file2}} {{file3}}`
 
-- Create a gzipped archive from a directory using relative paths:
+- [C]reate a g[z]ipped archive from a directory using relative paths:
 
 `tar czf {{target.tar.gz}} -C {{path/to/directory}} .`
 
-- Extract a (compressed) archive into the current directory:
+- E[x]tract a (compressed) archive [f]ile into the current directory:
 
 `tar xf {{source.tar[.gz|.bz2|.xz]}}`
 
-- Extract a (compressed) archive into the target directory:
+- [E]xtract a (compressed) archive [f]ile into the target directory:
 
 `tar xf {{source.tar[.gz|.bz2|.xz]}} -C {{directory}}`
```

Comment:
We can use the long format of `-C` to avoid explain that letter:
```suggestion
`tar xf {{source.tar[.gz|.bz2|.xz]}} --directory={{directory}}`
```

Reply:
Good idea.